forge-gui/res/cardsfolder/s/shattering_finale.txt

```diff
@@ -0,0 +1,7 @@
+Name:Shattering Finale
+ManaCost:1 B
+Types:Sorcery
+K:Casualty:1
+A:SP$ Effect | StaticAbilities$ PerpetualDebuff | RememberObjects$ Targeted | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | IsCurse$ True | Name$ Shattering Finale's Perpetual Effect | Duration$ Permanent | SpellDescription$ Target creature an opponent controls perpetually gets -0/-3.
+SVar:PerpetualDebuff:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ -0 | AddToughness$ -3 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This creature perpetually gets -3/-0.
+Oracle:Casualty 1\nTarget creature an opponent controls perpetually gets -3/-0.
```

Comment:
In both lines, should be -0/-3

No need for AddPower$ param since we aren't actually modifying power